hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/namenode/FSNamesystemLock.java

```diff
@@ -182,6 +192,7 @@ public void readUnlock(String opName,
     final long readLockIntervalMs =
         TimeUnit.NANOSECONDS.toMillis(readLockIntervalNanos);
     if (needReport && readLockIntervalMs >= this.readLockReportingThresholdMs) {
+      numReadLockLongHold.addAndGet(1);
```

Comment:
nit: you could use incrementAndGet. Don't have to fix it for me to commit it, just flagging for your consideration. 
https://docs.oracle.com/javase/8/docs/api/java/util/concurrent/atomic/AtomicLong.html#incrementAndGet--

Reply:
+1, LGTM. if you are making any changes, please let me know. Otherwise, I will leave for a little while for @arp7  or @xiaoyuyao  to take a quick look. Thank you for your work and contribution.

Reply:
Thank you for your review and suggestion. Using incrementAndGet is a better way. I fixed it.

Reply:
> nit: you could use incrementAndGet. Don't have to fix it for me to commit it, just flagging for your consideration.
> https://docs.oracle.com/javase/8/docs/api/java/util/concurrent/atomic/AtomicLong.html#incrementAndGet--

Hi @anuengineer , I fixed the problem. And I invited you to join the discussion on JIRA.